synapse/config/server.py

```diff
@@ -676,6 +676,10 @@ def read_config(self, config, **kwargs):
         ):
             raise ConfigError("'custom_template_directory' must be a string")
 
+        self.use_account_validity_in_account_status: bool = (
+            config.get("use_account_validity_in_account_status") or False
+        )
+
```

Comment:
Considering the MSC is unstable, should this be in the experimental/unstable part of the config?

Reply:
Experimental feels more to me like the place where we'd add toggles to enable entire MSC implems. Plus I'm not sure this qualifies as experimental in the same way other things in Synapse do - it's partly meant as a stopgap until Tchap stops relying on account validity (which should eventually be replaced by some LDAP-style central user directory). The long term plan is more to have it disappear when we don't need it anymore, rather than make it stable and expose it to the world.